VirtoCommerce.PricingModule.Data/VirtoCommerce.PricingModule.Data.csproj

```diff
@@ -70,8 +70,8 @@
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Data" />
     <Reference Include="System.Xml" />
-    <Reference Include="VirtoCommerce.Domain, Version=2.25.16.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\VirtoCommerce.Domain.2.25.16\lib\net461\VirtoCommerce.Domain.dll</HintPath>
+    <Reference Include="VirtoCommerce.Domain, Version=2.25.18.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Probably we could not update Domain version and use 2.25.16 (same in all places with updated version + manifest)

Reply:
Done